test/doConfigTestVm.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
This whole file was just renamed from `configTestVm.sh`.

No changes in here.

Reply:
something we need to think about as possibly another maint release in the future is doing argument validation further upstream.

Reply:
Yep. I started doing that by splitting up the original configTestVm.sh script but I backed out those changes because I thought it was making this stuff harder to work on/iterate/develop/grok, and I wanted to keep this PR as a strict "docker-ize it" change and not change too much.